docs/se/metrics/03_micrometer.adoc

```diff
@@ -0,0 +1,226 @@
+* Using the link:{javadoc-base-url-api}/MicrometerSupport.Builder.html[`MicrometerSupport.Builder`] class
+* Using configuration
+
+==== Using `MicrometerSupport.Builder`
+Use the `MicrometerSupport.Builder` class to set up Micrometer support however your application needs.
+
+The builder lets you:
+
+* Provide your own Micrometer meter registry configuration that `MicrometerSupport` uses to create a built-in meter
+registry, or
+* Instantiate a Micrometer meter registry yourself, configured however you want, and add it to the `MicrometerSupport`
+object's collection of meter registries
+
+[source,java]
+.Overriding defaults for built-in meter registries using `MicrometerSupport.Builder`
+----
+PrometheusConfig myPrometheusConfig = ...; // 1
+MicrometerSupport support = MicrometerSupport.builder()
+                .enrollBuiltInRegistry( // 2
+                        MicrometerSupport.BuiltInRegistryType.PROMETHEUS,
+                        myPrometheusConfig)
+                .build();
+----
+<1> Create the meter registry configuration however you need.
+<2> Enroll the `PROMETHEUS` built-in registry type with your custom configuration.
+
+
+==== Using Configuration
+To use configuration to control the selection and behavior of Helidon's built-in Micrometer meter registries
+include in your configuration a `metrics.micrometer.builtin-registries` section.
+
+[source,yaml]
+.Enroll Prometheus built-in meter registry using default configuration
+----
+metrics:
+  micrometer:
+    builtin-registries:
+      - type: prometheus
+----
+
+[source,yaml]
+.Enroll Prometheus built-in meter registry with non-default configuration
+----
+metrics:
+  micrometer:
+    builtin-registries:
+      - type: prometheus
+        prefix: myPrefix
+----
+
+Which configuration keys are valid in each `builtin-registries` entry depends on the type of Micrometer meter registry.
+Refer to the documentation for the meter registry you want to configure to find out what items apply to that registry
+type.
+
+Helidon does not validate the configuration keys you specify against the items defined by the corresponding
+meter registry configuration class.
+
+=== Enrolling other Micrometer meter registries
+To create your own registries and enroll them with `MicrometerSupport`, you need to:
+
+* Write a `Handler` +
++
+Each meter registry has its own way of producing output.
+Write your handler so that it has a reference to the meter registry it should use and so
+its `accept` method sets the payload in the HTTP response using the registry's mechanism for creating output.
+
+* Write a `Function` which accepts a `ServerRequest` and returns an `Optional<Handler>` +
++
+In general, your function looks at the request--the `Content-Type`, query parameters, etc.--to
+decide whether your handler should respond to the request.
+If so, your function should instantiate your `Handler` and return an `Optional.of(theInstance)`;
+otherwise, return your function should return `Optional.empty()`. +
```

Comment:
otherwise, your function should return ... (extra 'return'?)

Reply:
fixed